packages/calling/calling/src/Calling/types.ts

```diff
@@ -0,0 +1,37 @@
+import {CallingEventTypes} from '../Events/types';
+import {ICallHistory} from '../CallHistory/types';
+import {ICallSettings} from '../CallSettings/types';
+import {CallingClientConfig, ICallingClient} from '../CallingClient/types';
+import {IContacts} from '../Contacts/types';
+import {LoggerConfig} from '../Logger/types';
+import {ISDKConnector} from '../SDKConnector/types';
+import {IVoicemail} from '../Voicemail/types';
+import {Eventing} from '../Events/impl';
+
+interface ClientConfig {
+  calling?: boolean;
```

Comment:
Does boolean make sense here? You're setting it as optional, which means that this field doesn't need to be present in the object. So then setting this to false doesn't make sense right?